osu.Game/Overlays/MusicController.cs

```diff
@@ -316,6 +316,10 @@ private void prev()
 
         private void next(bool instant = false)
         {
+            // beatmapSets doesn't get populated until loading has completed.
```

Comment:
why is this here? if this is causing a crash or issue it should 100% be its own PR with own tests.

Reply:
this is causing a crash only on headless runs when going through the load process (since loading the `MusicController` can take a while). Can split this out.

Reply:
very highly doubt it's headless only. nothing stops a desktop run from also taking equally as long.

Reply:
Turns out this was unnecessary and was an issue in my test